dataproducts.tex

```diff
@@ -1,20 +1,20 @@
 
 \subsection{Catalog Server}
 
-Alternatively, an IDAC may find that hosting only the {\it Data Release} catalogs, and not the images, is sufficient for the scientific needs of its community. This will probably require the specific LSST database server \citedsp{LDM-135} and specific machines, and the deployment of the database system and the associated subset of data access services (DAX; e.g., web APIs, Qserv, \citeds{LDM-152}). The full {\tt Object} catalog, which contains one row per object with a volume of $\approx 20$ kilobytes per row, is estimated to contain about $40 \times 10^9$ objects (even in the first full-sky data release). Adding to this the full {\tt Source} and {\tt Forced Source} catalogs, which contain one row per measurement in each of the $\sim80$ visit images obtained per year, brings the total storage volume required up into the petabytes range, and will require a serious commitment of resources at the proposed IDAC. The evolution of catalog sizes over the 10-year LSST survey is depicted in \figref{fig:catvol}, from which it is evident that the catalog size for the final release is order $15$ petabytes. For more details on the row counts see the Key Numbers Page\footnote{\url{https://confluence.lsstcorp.org/display/LKB/LSST+Key+Numbers}}.
+Alternatively, an \gls{IDAC} may find that hosting only the {\it Data Release} catalogs, and not the images, is sufficient for the scientific needs of its community. This will probably require the specific \RO database server \citedsp{LDM-135} and specific machines, and the deployment of the database system and the associated subset of data access services (\gls{DAX}; e.g., web APIs, \gls{Qserv}, \citeds{LDM-152}). The full {\tt Object} catalog, which contains one row per object with a volume of $\approx 20$ kilobytes per row, is estimated to contain about $40 \times 10^9$ objects (even in the first full-sky data release). Adding to this the full {\tt Source} and {\tt Forced Source} catalogs, which contain one row per measurement in each of the $\sim80$ visit images obtained per year, brings the total storage volume required up into the petabytes range, and will require a serious commitment of resources at the proposed \gls{IDAC}. The evolution of catalog sizes over the 10-year \gls{LSST} is depicted in \figref{fig:catvol}, from which it is evident that the catalog size for the final release is order $15$ petabytes. For more details on the row counts see the Key Numbers Page\footnote{\url{https://confluence.lsstcorp.org/display/LKB/LSST+Key+Numbers}}.
 
```

Comment:
I have submitted a ticket to remove that confluence page from public view. The subsystem scientists plan to do some more work on it. The page that should be referenced is the public facing one: https://www.lsst.org/scientists/keynumbers

Reply:
ok thats easy